scripts/build_artifacts.sh

```diff
@@ -11,17 +11,24 @@
 #  distributed under the License is distributed on an "AS IS" BASIS,
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
-#  limitations under the License.
+# limitations under the License.
 
-# Run this script as follows:
-# DATABRICKS_URI=https://<your-databricks-instance>.azuredatabricks.net ./setup_cli.sh
+set -o errexit
+set -o pipefail
+set -o nounset
+# Uncomment this line to see each command for debugging (careful: this will show secrets!)
+# set -o xtrace
 
-set +e
 
-# Resource ID here is the same for all Databricks workspaces
-# (see https://learn.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/aad/service-prin-aad-token)
-token_response=$(az account get-access-token --resource 2ff814a6-3304-4ab8-85cb-cd0e6f879c1d)
-DATABRICKS_AAD_TOKEN=$(jq .accessToken -r <<< "$token_response")
-export DATABRICKS_AAD_TOKEN
+ROOT="/workspaces/FlowEHR"
```

Comment:
may be safer to use a path relative to this file i.e.

```bash
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
PIPELINE_DIR="${SCRIPT_DIR}/../transform/pipelines" 
```
? for clarity and to decouple from the devcontainer folder structure
(here and similarly in other scripts)

Reply:
good point, will do that